seeds/data/cases.json

```diff
@@ -143,8 +143,8 @@
       "action": "grant",
       "subject": "5b7bad13-f34b-4959-bd08-c6067ae2fcdd",
       "deadline": {
-        "standard": "2020-07-24",
-        "extended": "2020-08-14",
+        "standard": "2020-07-27",
+        "extended": "2020-08-17",
```

Comment:
The dates on this seed (and related activity) did not make sense for the submission date of `2020-06-01`. This is the correct deadline.